src/AudioManagerImpl.js

```diff
@@ -161,7 +162,7 @@ export default class AudioManagerImpl implements AudioManager {
             // As we use a single Sampler grade, our best option for panning is
             // to pan all sounds.  We can discuss adjusting this once we have
             // multiple sound-producing elements in the environment.
-            const panningLevel = Math.min(1, Math.max(-1, (0.1 * characterState.xPos)));
+            const panningLevel = ((characterState.xPos / sceneDimensions.getWidth()) * 2) - 1;
```

Comment:
The `panningLevel` calculation isn't quite symmetrical. The `xPos` varies from 1-26 inclusive.

- At `xPos` of 1, `panningLevel` is -0.92
- At `xPos` of 26, `panningLevel` is 1

We'd want a calculation like:

    const panningLevel = ((characterState.xPos - minX) / (maxX - minX) * 2) - 1;

Where `minX` and `maxX` are from the `SceneDimensions`. Unfortunately, right now `SceneDimensions` will not give us the right min X and max X. It may be too risky to rework `SceneDimensions` at this point in the 0.7 timeline. I have another thought on panning, which I will leave as a separate comment.

Reply:
In the new coordinate system, the `SceneDimensions` constructor should really now be:

    constructor(minX: number, maxX: number, minY: number, maxY: number)

So we can construct with:

    new SceneDimensions(1, 26, 1, 16);

But as I mentioned above, it may be too risky to rework at this stage.

Reply:
I hard-coded the current minimum in an adjusted calculation of the midpoint, so that the panning is symmetrical.

I also left a TODO in the relevant area about using minX and maxX.